requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+black==19.10b0
+pylint==2.4.4
+mypy==0.761
+-r scripts/requirements.txt
```

Comment:
for developers they will do `pip install -r requirements-dev.txt` so they can run the format checks locally